djot_parser/examples/todo-list.html

```diff
@@ -1,11 +1,14 @@
 <ul class="task-list">
-<li class="unchecked">
```

Comment:
Let's add one more test where checkbox item list will have some non-paragraph content inside like:
```
- [ ] unchecked

 * 1
 * 2
 * 3
- [x] checked
- [ ] finish
```

Reply:
One other change to consider are nested paragraphs like the djot.js test: https://github.com/dcampbell24/djot.js/blob/22470195eb0dc1fc942b34b7c8d95b23597c094a/test/task_lists.test

I added the test from `djot.js` verbatim. While the `*`-based lists are currently failing, I can address that in a future PR (https://htmlpreview.github.io/?https://github.com/jgm/djot/blob/master/doc/syntax.html#list-item)